### PR TITLE
Another initial setup

### PR DESCRIPTION
**Pull request by Kushagra-0801**
Based on the PR by @VikramjeetD 
Used the `import` syntax of ES6 and removed a lot of the dependencies

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    "debug": "^4.1.0",
+    "dotenv": "6.2.0",
+    "express": "^4.17.1",
+    "express-validator": "^6.4.0",
+    "mongoose": "^5.8.11",
+    "node-sass": "^4.5.3",
+    "node-sass-middleware": "^0.11.0",
+    "react": "^16.6.3",
+    "react-dom": "^16.6.3"
+  },
```
Comment (VikramjeetD) [inline package.json:21]: `bcryptjs` and `jsonwebtoken` were for implementing staff auth functions (encryption and stuff). 
Have any alternative in mind?

Comment (Kushagra-0801) [reply package.json:21]: The login module will be provided by the TA na? Let's see if that one won't have the stuff for staff login included

Comment (VikramjeetD) [reply package.json:21]: Iirc, Sir said the **google** login implementation would be provided by the TA. Anyways, the rest looks good.

**Review by VikramjeetD — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (Kushagra-0801) [reply package.json:21]: I mean the google login would also probably use some other library. So we can just reuse that one. Or we can add these later if it doesn't work out

**Pull request merged**